src/storage/QueryEdgePropsProcessor.cpp

```diff
@@ -62,7 +62,7 @@ void QueryEdgePropsProcessor::process(const cpp2::EdgePropRequest& req) {
     RowSetWriter rsWriter;
     std::for_each(req.get_parts().begin(), req.get_parts().end(), [&](auto& partE) {
         auto partId = partE.first;
-        kvstore::ResultCode ret;
+        kvstore::ResultCode ret = kvstore::ResultCode::SUCCEEDED;
```

Comment:
why use `SUCCEEDED` to set ret ? Is it really necessary ?

Reply:
It's to suppress compilers warning of uninitialized variables